File: practica/patrones-de-diseño/java8/command/switchLampara/Lampara.java
/**
 * Receptor - Lampara
 */
public class Lampara {
  private boolean encendido;
  
  public void accionEncender(){
    encendido = true;
    System.out.println("Lampara encendida");
  }
  
  public void accionApagar(){
    encendido = false;
    System.out.println("Lampara apagada");
  }
}

File: practica/patrones-de-diseño/typescript/command/lampara/src/Lampara.ts
export class Lampara{
    private encendido:boolean = false

    public getEncendido():boolean{
        return this.encendido
    }

    public accionEncender(): void{
        this.encendido = true
    }

    public accionApagar(): void{
        this.encendido = false
    }
}
